book/pipeline-service.md

```diff
@@ -1,29 +1,27 @@
 # Pipeline Service
 
-Pipeline Service provides Tekton APIs and services to kcp workspaces.
-When a workspace binds to the Pipeline Service APIExport, it has access to the following Tekton APIs:
+Pipeline Service provides a SaaS for pipelines. It leverages:
 
-- `Task`
-- `Pipeline`
-- `PipelineRun`
-- `Repository` (for [Pipelines as Code](https://pipelinesascode.com))
+- Kubernetes / OpenShift for the compute
+- Tekton Pipelines, Results and friends for the core of the service
+- OpenShift GitOps / Argo CD, Pipelines as Code for managing the infrastructure
 
-The following features are also provided:
+The following features are provided:
 
 - Signing and attestation of `TaskRuns` with [Tekton Chains](https://tekton.dev/docs/chains/)
-- Archiving of `PipelineRuns` and `TaskRuns` with [Tekton Results](https://tekton.dev/docs/results/)
+- Archiving/Pruning of `PipelineRuns` and `TaskRuns` with [Tekton Results](https://tekton.dev/docs/results/)
 - Integrations with GitHub, Gitlab, and Bitbucket with [Pipelines as Code](https://pipelinesascode.com) (PaC).
 
 ## Architecture
 
-Tekton controllers are deployed directly on the workload cluster.
-The kcp syncer for the workload cluster is configured to sync `Pipeline`, `Task`, and `PipelineRun` objects.
-When a `PipelineRun` is created on kcp, it is synced to the workload cluster whose controllers then create `TaskRun` objects and resulting `Pods`.
-The Chains and Results controllers then work with the synced `PipelineRun` and generated `TaskRun` objects to perform their respective actions.
+Pipeline Service is deployed onto the cluster alongside other AppStudio components. The following form the core architecture of Pipeline Service:
 
-Like upstream Tekton, Pipelines as Code (PaC) also runs controllers directly on workload clusters.
-Unlike other Tekton components, PaC also exposes a `Route`/`Ingress` that allows it to receive webhook events from source control repositories.
-To sent traffic from kcp to the workload cluster, Pipeline Service will deploy a "gateway" service that acts as a proxy to PaC on the workload cluster.
+1. _OpenShift Pipelines:_ It enables the creation and management of Pipelines for user Applications.
+2. _Pipelines as Code (PaC):_ An OpenShift Pipelines component that enables receiving webhook events from version control systems. User would install a Github App to configure this.
+3. _Spray Proxy:_ Forwards requests to all clusters in the environment once an event is received on PaC. It is a reverse proxy that broadcasts to multiple backends.
+4. _Tekton Results:_ Responsible for storing logs and pruning PipelineRuns. Utilises AWS RDS as the backend database.
```

Comment:
The text "Utilises AWS RDS as the backend database." assumes that we install stonesoup on AWS. What about local installation, which DB will then be used ?

Reply:
```suggestion
4. _Tekton Results:_ Responsible for storing logs and pruning PipelineRuns. Utilises AWS RDS as the backend database for our production instance.
```

Reply:
@ralphbean I assume that all the information on this page (and other components) are referring to the production instance unless otherwise specified. Do you think its necessary to mention that its production? Doing that will force us to distinguish between production & staging and development for every scenario.

@cmoulliard 
- No, installing/deploying Stonesoup on AWS is different from configuring AWS RDS DBs. In the Production instance which our users will access, there is no "local db". Our clusters are configured to use RDS and all the logs are stored there.
- However, if a user wants to try Stonesoup in preview mode on their cluster by running our scripts, then we have a mechanism in Pipeline Service that would configure PostgreSQL locally.
But again, as I have mentioned in the previous point, all the details on this page is strictly referring to the Production clusters.

Please let me know your thoughts.

Reply:
Personally, I'm ok to omit the new specific clause. I was trying to address @cmoulliard's concern. Let's leave it out of this change and if we want to take another pass to specify differences between prod/staging/local dev, let's do it separately.

Reply:
Question: If this book will become the upstream documentation book that users will read to play with AppStudio, why are you then referring to production instances which are then part of the downstream product RHTAP and our Saas offering ? @bnallapeta 

To clarify the discussion about AppStudio upstream, should we then instead describe what the different environments refer to (dev/staging/prod) ? Where AppStudio could be deployed (AWS, on-premises ?, ....) and the backend systems needed such as: PostgreSQL, https://min.io/, etc ?  

@ralphbean @adambkaplan @bnallapeta

Reply:
@cmoulliard My understanding of this repo is that it will act as a source of truth for our documentation on AppStudio. The general architecture, diagrams, tests, security, ADRs etc form this repo. In that context, I don't think this repo is intended for a user to "try" AppStudio. For that, they would need to go to [infra-deployments](https://github.com/redhat-appstudio/infra-deployments) repo.

So, whenever we describe architecture of a component or AppStudio itself, we refer to the production instance. (Also, @ralphbean stated that the differentiation between dev/staging/prod can be taken up in a separate PR)